std_msgs/Header header
float64[4] refletancia
